torch/csrc/tensor/python_tensor.cpp

```diff
@@ -5,10 +5,13 @@
 #include <sstream>
 
 #include "torch/csrc/assertions.h"
+#include "torch/csrc/Dtype.h"
+#include "torch/csrc/DynamicTypes.h"
 #include "torch/csrc/Exceptions.h"
 #include "torch/csrc/autograd/variable.h"
 #include "torch/csrc/autograd/python_variable.h"
 #include "torch/csrc/autograd/generated/VariableType.h"
+#include "torch/csrc/autograd/utils/wrap_outputs.h"
```

Comment:
What's this doing here?

Reply:
To return the dtype we call:
```
return torch::autograd::utils::wrap(self->dtype);
```

which just does a Py_INCREF(dtype); it seemed nice to reuse that function but I'm fine changing it.

Reply:
Oh, that seems fine. I was confusing it with the function `wrap_outputs` in torch/csrc/autograd/functions/utils.h